nion/swift/test/DisplayPanel_test.py

```diff
@@ -2564,6 +2564,41 @@ def test_interval_zoom(self):
                 self.assertAlmostEqual(axes.drawn_left_channel, (min(interval) - padding) * 1024)
                 self.assertAlmostEqual(axes.drawn_right_channel, (max(interval) + padding) * 1024)
 
+    def test_stacked_lineplot_interval_zoom(self):
```

Comment:
I'm curious why this test passes when the code is mixing intensity/dimensional calibrations. It's probably worth revisiting this test before making further changes to ensure it is testing the right thing.

It's easy to construct data that fails: just make one line plot 16x1 with a value of 1 (or random) and intensity calibration of 1.0 scale and units of 'z'. Then make another 16x1 with value of 1 (or random) and intensity calibration of 8.0 scale and units of 'z'. Drag the 2nd onto the first, make an interval, and view to interval. It will fail.